### PR TITLE
Artciukas test

### PR DESCRIPTION
**Pull request by artciukastest**
Just message

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+my-created.txt
```
Comment (artciukas) [inline .gitignore:1]: Good

File: main.py [hunk 2]
```diff
@@ -0,0 +1 @@
+print("Hi")
```
Comment (artciukas) [inline main.py]: Please change print to "Welcome"

**Review by artciukas — COMMENTED** [review]
finish comment

File: .gitignore (hunk 1 above)
Comment (artciukastest) [reply .gitignore:1]: Thanks

File: main.py [hunk 3]
```diff
@@ -1 +1 @@
-print("Hi")
\ No newline at end of file
+print("Welcome")
```
Comment (artciukas) [inline main.py:1]: Now its ok

**Review by artciukas — APPROVED** [review]

**Pull request merged**